packages/astro/src/core/build/vite-plugin-css.ts

```diff
@@ -239,9 +239,10 @@ export function rollupPluginAstroBuildCSS(options: PluginOptions): VitePlugin[]
 						if (output.type === 'asset') {
 							if (output.name?.endsWith('.css') && typeof output.source === 'string') {
 								const cssTarget = settings.config.vite.build?.cssTarget;
+								const minify = settings.config.vite.build?.minify === false ? false : true;
```

Comment:
Tiny nit but I think this could be done shorter 😬 
```suggestion
								const minify = settings.config.vite.build?.minify !== false
```